zmq.hpp

```diff
@@ -313,6 +313,19 @@ inline int poll(std::vector<zmq_pollitem_t> &items,
 inline int poll(std::vector<zmq_pollitem_t> &items, long timeout_ = -1)
 {
     return poll(items.data(), items.size(), timeout_);
+}		 
+
+template<std::size_t SIZE>
+inline int poll(std::array<zmq_pollitem_t, SIZE>& items, long timeout_ = -1)
+{
+    return poll(items.data(), items.size(), timeout_);
+}
+
```

Comment:
Please remove this. The corresponding overload for `std::vector` and a `long` timeout is already deprecated. We shouldn't add a new one.

Reply:
Not a problem removing that overload but I don't see it deprecated for `std::vector` on master

Reply:
Oh, you're right. I confused that with the `const` overload. Anyway, it *should* be deprecated, and so it would be great not to add a similar overload :)

Reply:
I amended the PR removing the said overload. Additionally I marked the std::vector one, taking long, as deprecated